let generate-cran-packages.R complete without needing to download the whole CRAN

aavogt opened the pull request:
CRAN doesn't include old versions at the urls that nix is looking at. So nixpkgs cran packages break pretty quickly. `generate-cran-packages.R` completes much quicker with this change.

aavogt committed:
let generate-cran-packages.R read checksums from cran-packages.nix

this lets "Rscript generate-cran-packages.R > cran-packages.nix"
complete much quicker, because we only have to download packages
with new versions.

aavogt committed:
update cran packages

peti commented:
Have you checked that all entries in `cran-packages.nix` still evaluate, i.e. by running `nix-env -qaP -A rPackages`?

aavogt commented:
On my system  `nix-env -qaP -A rPackages` says: `error: attribute ‘rPackages’ in selection path ‘rPackages’ not found`

But in my config.nix I build a fair number of packages from there.
```
    myREnv = pkgs.rWrapper.override {
      packages = with pkgs.rPackages; [
akima AlgDesign crs data_table directlabels knitr flexmix
            gdata ggdendro ggmap dplyr ggplot2 scam deSolve
            plyr polynom reshape2
            lattice lpSolve lpSolveAPI
            rootSolve phreeqc ] ;};
```

I can replicate that MASS-7.3-39 checksum change travis found.

peti commented:
What does `nix-env -qaP | head` say on your machine?

aavogt committed:
remove some packages marked broken for cran-packages.nix

those packages were removed from cran (and cran-packages.nix)

aavogt committed:
CRAN MASS_7.3-39 was uploaded twice, so use the newer checksum

The two dates are:

Packaged: 2015-02-20 08:29:38 UTC; ripley
Packaged: 2015-02-24 10:34:20 UTC; ripley

There were with changes to tests/Examples/MASS-Ex.Rout.save,
but the rest of the code seems unaffected.
